src/scss/components/button/mixins/_hxButton.scss

```diff
@@ -184,7 +184,7 @@
 @mixin hxButton--tertiary {
   background-color: transparent;
   border: 0;
-  color: $cyan-900;
+  color: $cyan-500;
```

Comment:
Please update to the new `v2.0` color variables:

https://github.com/HelixDesignSystem/helix-ui/blob/eb547a4294af4f768294ffef085cfa58136718c1/src/scss/_vars.scss#L1-L118

Reply:
Changes have been done, Please review